outputs.tf

```diff
@@ -45,6 +45,26 @@ output "environment_id" {
   description = "string ||| The ID of the this Elastic Beanstalk Environment"
 }
 
+output "log_provider" {
+  value       = "cloudwatch"
+  description = "string ||| "
+}
+
+output "log_group_name" {
+  value       = module.logs.name // TODO: fix this. how do I get the log group name that's created from Terraform?
```

Comment:
I'm not sure yet how to get access to the CloudWatch Log Group created from Beanstalk. I'm still researching that.

Reply:
I added support for streaming multiple log groups from the CLI in https://github.com/nullstone-io/nullstone/pull/176

To utilize this, the output for `log_group_name` should be `"/aws/elasticbeanstalk/${aws_elastic_beanstalk_application.this.name}/*"`